src/components/Payment/Payment.tsx

```diff
@@ -69,6 +76,63 @@ const Payment = ({
   const breakpoint = useBreakpoint();
   const isMobile = breakpoint === Breakpoint.xs;
 
+  const { offers } = useOffers();
```

Comment:
The same applies here. The `useOffers` hook is also store dependent and should not be used in components directly.

Reply:
I'm not sure how else to approach this after giving it some thought. Any ideas?

Reply:
You could create a `<PaymentContainer />`. That would also simplify the `User` screen a bit.

Reply:
Please take a look at the most recent commits and see if that's what you had in mind